query/src/sql/sql_parser.rs

```diff
@@ -854,23 +854,108 @@ impl<'a> DfParser<'a> {
         Ok(DfStatement::DropStage(drop))
     }
 
+    fn parse_udf_parameters(&mut self) -> Result<Vec<String>, ParserError> {
+        let mut params = vec![];
+        let mut found_right_paren = false;
+        let mut expect_next_param = false;
+
+        self.parser.expect_token(&Token::LParen)?;
+        loop {
+            let next_token = self.parser.next_token();
+            if next_token == Token::EOF || next_token == Token::SemiColon {
+                break;
+            }
+            if next_token == Token::RParen {
+                found_right_paren = true;
+                break;
+            }
+
+            match next_token {
+                Token::Word(Word {
```

Comment:
`self.parser.parse_comma_separated` can be addressed.

Reply:
Checked this method, it parses a comma-separated list of 1+ items, we need to handle zero param as well.